content/post/blackboard_submission_service.md

```diff
@@ -0,0 +1,105 @@
++++
+date = "2015-01-19T19:50:00+10:00"
+draft = true
+title = "Blackboard Extensions and the Submission Service"
+description = "Stepping through the process for adding the required fields to your bb-manifest file."
+keywords = ["blackboard"]
++++
+
+# Blackboard Extensions and the Submission Service
+
+Extensions are one of many Blackboard APIs, and like the rest of them, are under-documented or have very limited implementation details. This post will walk through the process for implementing one such extension provided by Blackboard, namely the submission service. The submission service provides event handlers for submittable items (assignments, tests, et.al). For example when a student submits an assignment, one of the handlers in this extension fires. So let's jump into the code!
+
+## Quick Setup
+
+Start by grabbing the jar file, it can be obtained from the Blackboard development vm. For the submission service, the jar file is located at `/usr/local/blackboard/content/vi/BBLEARN/plugins/bb-submission-services/webapp/WEB-INF/libext/submission-services-api-1.1-SNAPSHOT.jar`. Move this file onto the class path for your building block.
```

Comment:
We need to mention that this jar should be added to the compile time classpath and they need to make sure that it doesn't end up in their WAR. Maybe mention that the changes to the bb-manifest that are coming up will ensure it's added to the runtime classpath.